demisto_sdk/commands/common/git_util.py

```diff
@@ -795,3 +795,14 @@ def get_all_changed_files(
             get_only_current_file_names=True,
         )
         return modified_files.union(added_files).union(renamed_files)
+
+    def _is_file_git_ignored(self, file_path: str) -> bool:
+        """return wether the file is in .gitignore file or not.
+        Args:
+            file_path (str): the file to check.
+        Returns:
+            bool: True if the file is ignored. Otherwise, return False.
+        """
+        if self.repo.ignored(file_path):
+            return True
+        return False
```

Comment:
```suggestion
        return self.repo.ignored(file_path):
```

Reply:
repo.ignored returns a list which is empty if the file isn't ignored, I want to return bool.. changing to:
`return True if self.repo.ignored(file_path) else False`

Reply:
better do `bool(self.repo.ignored(file_path))`

Reply:
cool, didn't know this